docs/modules/user-guide/partials/proc_refering-to-a-parent-devfile-in-a-devfile.adoc

```diff
@@ -2,17 +2,22 @@
 = Refering to a parent devfile in a devfile
 
 [role="_abstract"]
-It is possible to refer to another devfile as a parent to a given devfile. If a parent is specified, the devfile will inherit all behaviour from the parent. Part of the content of the parent can be overridden in the child devfile. That allows refering a devfile (the parent) from a distinct devfile and makes it possible to reuse in multiple devfiles the same parent (the stack).
+This section describes how to designate a parent devfile to a given devifle. If you designate a parent devfile, the given devfile inherits all its behavior from its parent.  Still, you can use the child devfile to override certain content from the parent devfile. If you override the correct content, you can reuse the same parent devfile in multiple other devfiles. If you do reuse a parent devfile, the parent turns into a stack that is used in multiple other devfiles.
 
 .Prerequisites
 
-* xref:adding-schema-version-to-a-devfile.adoc[]
-* xref:adding-a-name-to-a-devfile.adoc[]
+* Define the schemaVersion to v2.x. If you need to define your schemaVersion, go to xref:adding-schema-version-to-a-devfile.adoc[].
+* Define a name for the devfile. If you need to define a name for your devfile go to xref:adding-a-name-to-a-devfile.adoc[].
 
 
 .Procedure
 
-* A parent devfile can be referenced in 3 different ways: `registry`, `uri` or `kubernetes`.
+You can refer to a parent devfile in three different ways:
+
+* registry
```

Comment:
Shouldn't these elements be formatted like they were before?

Reply:
Eva said it's good to give context for the prereqs and I thought it did help users know if they should even bother clicking the links. Do you think the sentence, "define the...if you need to define..." is unhelpful? Or can it be kept?

Reply:
I just meant that elements such as `registry` should use the same formatting (monospace). :)